proto/cosmos/group/v1/tx.proto

```diff
@@ -306,11 +302,20 @@ message MsgSubmitProposal {
   // whether it should be executed immediately on creation or not.
   // If so, proposers signatures are considered as Yes votes.
   Exec exec = 5;
+
+  // title is the title of the proposal.
+  //
+  // Since: cosmos-sdk 0.47
+  string title = 6;
```

Comment:
One thing I forgot to notice in the original PR: we need to bump gov and group's ConsensusVersion

Reply:
good point!

Reply:
Checking now, gov has been bumped with params change, so we are good!